test/unit/lodash.spec.js

```diff
@@ -0,0 +1,690 @@
+                var array = params[0];
+                var countString = 0;
+                var countInt = 0;
+
+                array.forEach(function (i) {
+                    if (typeof i === 'string') {
+                        countString++;
+                    } else if (typeof i === 'number') {
+                        countInt++;
+                    }
+                });
+                expect(array.length).to.eql(6);
+                expect(countInt).to.eql(3);
+                expect(countString).to.eql(3);
+            });
+            it('should group element of array by length', function () {
+                expect(_.groupBy.apply(_, datasets.groupBy2())).to.eql({
+                    3: ['xyz', 'xyz'],
+                    5: ['combi'],
+                    undefined: [2, 4, 5]
+                });
+            });
+        });
+    });
+
+    describe('includes', function () {
+        describe('incloudes function', function () {
+            var params = datasets.includes();
+            var obj = params[0];
+            var predicate = params[1];
+
+            it('should match types of passing elements', function () {
+                var element1 = params[0] instanceof Object;
+                var element2 = params[1] instanceof Array;
+
+                expect(element1).to.eql(true);
+                expect(element2).to.eql(true);
+                expect(params.length).to.eql(2);
+                expect(params[1].length).to.eql(2);
+            });
+            it('should contain an object with provided properties', function () {
+                expect(Object.keys(obj).length).to.eql(4);
+                expect(_.has(obj, 'firstName')).to.eql(true);
+                expect(_.has(obj, 'lastName')).to.eql(true);
+                expect(_.has(obj, 'users')).to.eql(true);
+                expect(_.has(obj, 'running')).to.eql(true);
+                expect(typeof obj.firstName).to.eql('string');
+                expect(typeof obj.lastName).to.eql('string');
+                expect(typeof obj.users).to.eql('number');
+                expect(typeof obj.running).to.eql('boolean');
+            });
+            it('should include first value in the object', function () {
+                expect(_.includes.apply(_, [obj, predicate[0]])).to.eql(true);
+                expect(_.includes.apply(_, [obj, predicate[1]])).to.eql(false);
+            });
+        });
+        describe('includes2 function', function () {
+            var params = datasets.includes2();
+
+            it('should match types of passing elements', function () {
+                var element1 = params[0] instanceof Array;
+                var element2 = typeof params[1];
+
+                expect(element1).to.eql(true);
+                expect(element2).to.eql('string');
+                expect(params.length).to.eql(2);
+            });
+            it('should include \'stabli\' word in array of words', function () {
```

Comment:
To be honest with you even I had hard time reading this task. So much logic and even inside loop made it very lengthy, futhermore scenario does not include proper explanation. Please do split it up into smaller tasks by putting describe and inside it tasks for specific index.

I was also able to use any word of my choosing instead of stabli, please fix that too without indicating answer for user.